src/components/table/BadgesCellRenderer.tsx

```diff
@@ -24,19 +26,30 @@ const BadgesCellRenderer = ({
   tagList,
 }: Props): ((arg: ChildCompProps) => JSX.Element) => {
   const ChildComponent = ({ data: item }: ChildCompProps) => {
+    const { t } = useBuilderTranslation();
     const { data: itemTags } = hooks.useItemTags(item.id);
 
     const isHidden = isItemHidden({ tags: tagList, itemTags });
     const isPublic = isItemPublic({ tags: tagList, itemTags });
     const isPublished = isItemPublished({ tags: tagList, itemTags });
     const isPinned = Boolean(item?.settings?.isPinned);
+    const isCollapsible = Boolean(item?.settings?.isCollapsible);
+    const showChatbox = Boolean(item?.settings?.showChatbox);
 
     return (
       <ItemBadges
         isPinned={isPinned}
+        isPinnedTooltip={t(BUILDER.STATUS_TOOLTIP_IS_PINNED)}
         isHidden={isHidden}
+        isHiddenTooltip={t(BUILDER.STATUS_TOOLTIP_IS_HIDDEN)}
         isPublic={isPublic}
+        isPublicTooltip={t(BUILDER.STATUS_TOOLTIP_IS_PUBLIC)}
         isPublished={isPublished}
+        isPublishedTooltip={t(BUILDER.STATUS_TOOLTIP_IS_PUBLISHED)}
+        isCollapsible={isCollapsible}
+        isCollapsibleTooltip={t(BUILDER.STATUS_TOOLTIP_IS_COLLAPSIBLE)}
+        showChatbox={showChatbox}
+        showChatboxTooltip={t(BUILDER.STATUS_TOOLTIP_SHOW_CHATBOX)}
```

Comment:
IMO this shows that the current API is a bit clunky, it means that adding adding or removing badges requires changes in both the provider and the consumer.

In the future, would it make more sense to have something like:
```jsx
<ItemBadges>
   {isPinned && <Badge icon={pinIcon} tooltip={t(BUILDER.STATUS_TOOLTIP_IS_PINNED)} />}
   {isHidden && <Badge icon={hideIcon} tooltip={t(BUILDER.STATUS_TOOLTIP_IS_HIDDEN)} />}
   ... // etc
</ItemBadges>
```